app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageViewModel.kt

```diff
@@ -91,7 +91,7 @@ class ZimManageViewModel @Inject constructor(
   private val dataSource: DataSource,
   private val connectivityManager: ConnectivityManager,
   private val sharedPreferenceUtil: SharedPreferenceUtil
-) : ViewModel() {
+) : AndroidViewModel(context) {
```

Comment:
Passing context to viewModels is a crime! It is an open invitation to memory leaks! Also `AndroidViewModels` are never supposed to be used as they are quite tightly couple with android arch and ain't flexible.